### ISSUE TITLE
Alter language used in settings

### ISSUE THREAD

**Issue by KairuByte** [Phalcode/gamevault-app#193]
The use of "app" in the settings feels off, like it's meant to be generic. Instances of it should likely be changed to "GameVault" with any grammatical changes needed. (ex. "Start GameVault in background")

The use of "autostart" in the settings is a little ambiguous. "App autostart" should likely be changed to something more descriptive like "Run GameVault when my system boots" or similar.

"Load library on startup" is a little less ambiguous, but would likely do well with a change to "Load server library when GameVault starts".

"background" is again ambiguous. "Start GameVault in background" should likely be "Start GameVault in system tray"

tl;dr I am suggesting the following wording based on what is currently there:

- Start GameVault in system tray
- Load game library when GameVault starts
- Run GameVault when my system boots

**Comment by Alfagun74** [comment]
I think i will join all these Settings related Tickets into a single one.

**Comment by Alfagun74** [comment]
Merged into #198
